.github/workflows/check-migrations.yml

```diff
@@ -113,31 +172,45 @@ jobs:
           components: "rust-src"
           toolchain: "${{env.RUST_STABLE_VERSION}}"
 
+      - name: Download Snapshot
+        uses: actions/cache@v4
+        with:
+          path: snapshot.raw
+          key: try-runtime-snapshot-${{ matrix.runtime.name }}
+
+      - name: Verify snapshot exists
+        run: |
+          if [ ! -f "snapshot.raw" ] || [ ! -s "snapshot.raw" ]; then
+            echo "No valid snapshot found. This job requires pre-generated snapshots from the nightly job."
+            echo "Please wait for the nightly snapshot generation or run it manually via workflow_dispatch."
+            exit 1
+          else
+            echo "Using snapshot from $(date -r snapshot.raw)"
+          fi
+
       - name: Run ${{ matrix.runtime.name }} Runtime Checks
         #uses: "paritytech/try-runtime-gha@v0.2.0"
         env:
           EXTRA_FLAGS: ${{ env.EXTRA_FLAGS }}
           CHECKS: ${{ env.CHECKS }}
         run: |
-          cargo install -q --git https://github.com/paritytech/try-runtime-cli --tag v0.8.0 --locked && try-runtime --version
-
-          cargo build --profile production -p ${{ matrix.runtime.package }} --features try-runtime -q --locked
+          try-runtime --version
```

Comment:
Isn't `try-runtime` installed only for `generate-snapshots`?
Don't we need to pass it also via "Download Snapshot" or something?